src/ray/object_manager/push_manager.cc

```diff
@@ -76,7 +76,9 @@ void PushManager::ScheduleRemainingPushes() {
 }
 
 void PushManager::RecordMetrics() const {
-  // TODO(sang): Add metrics.
+  ray::stats::STATS_push_manager_push_count.Record(NumPushesInFlight(), "InFlight");
```

Comment:
There's no breakdown. I will remove the label in this case

Reply:
Actually thinking more about it, isn't it better having a single label than re-name the metric in case we will add new labels in the future? (It is also good for backward compatibility. Though it is not requirement, it is a good property to keep since whenever we change the metric name, we should modify our Grafana dashboard).